extensions/resource-deployment/src/ui/model.ts

```diff
@@ -32,6 +33,24 @@ export class Model {
 		return value === undefined ? defaultValue : value === 'true';
 	}
 
+	public getCodeCellContentForNotebook(): string[] {
+		const statements: string[] = Object.keys(this.propValueObject)
+			.filter(propertyName => propertyName.startsWith(NoteBookEnvironmentVariablePrefix))
+			.map(propertyName => {
+				const value = this.escapeForNotebookCodeCell(this.getStringValue(propertyName, ''));
+				const regex = new RegExp(`^${NoteBookEnvironmentVariablePrefix}`);
```

Comment:
This regex can just be a const declared outside the class - no reason to make a separate one for every line.